scss/config/_fonts.scss

```diff
@@ -8,14 +8,6 @@
 ///   Accoutrement Type
 
 
-// Font Formats
-// ------------
-/// The font-formats that we want to import
-///
-/// @group config-fonts
-$font-formats: 'woff' 'ttf';
```

Comment:
Since I removed this, not sure if we need to add it back in to the webfont Sass maps?

Reply:
We're not using any imported fonts, so this is fine.